runtimes/eden/src/pallets_util.rs

```diff
@@ -173,9 +173,9 @@ impl pallet_contracts::Config for Runtime {
 	type DeletionQueueDepth = DeletionQueueDepth;
 	type DeletionWeightLimit = DeletionWeightLimit;
 	type Schedule = MySchedule;
-	type CallStack = [Frame<Self>; 31];
+	type CallStack = [Frame<Self>; 5];
 	type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
-	type MaxCodeLen = ConstU32<{ 128 * 1024 }>;
+	type MaxCodeLen = ConstU32<{ 123 * 1024 }>;
```

Comment:
Any reasons why this is getting lowered?

Reply:
I follow the contracts system parachain and the fact that they have gone for a lower value here could be due to some safety analysis. We'd better go for the lower and increase it in the future when we have evaluated all the cons and pros.

Reply:
Yes of course - I was wondering as to _why_ they did the change, not necessarily _if_ we should do it

Reply:
Looks like Parity's team figured it would be appropriate to document the limits and safe values in the not so well known `integrity_test` function. https://paritytech.github.io/substrate/master/src/pallet_contracts/lib.rs.html#323-386